ApiDemos/kotlin/app/src/gms/java/com/example/kotlindemos/StreetViewPanoramaOptionsDemoActivity.kt

```diff
@@ -0,0 +1,117 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.example.kotlindemos
+
+import android.os.Bundle
+import android.view.View
+import android.widget.CheckBox
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.google.android.gms.maps.StreetViewPanorama
+import com.google.android.gms.maps.SupportStreetViewPanoramaFragment
+import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.StreetViewSource
+
+
+/**
+ * This shows how to create an activity with static streetview (all options have been switched off)
+ */
+class StreetViewPanoramaOptionsDemoActivity : AppCompatActivity() {
+    private var streetViewPanorama: StreetViewPanorama? = null
+    private lateinit var streetNameCheckbox: CheckBox
+    private lateinit var navigationCheckbox: CheckBox
+    private lateinit var zoomCheckbox: CheckBox
+    private lateinit var panningCheckbox: CheckBox
+    private lateinit var outdoorCheckbox: CheckBox
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.street_view_panorama_options_demo)
+        streetNameCheckbox = findViewById(R.id.streetnames)
+        navigationCheckbox = findViewById(R.id.navigation)
+        zoomCheckbox = findViewById(R.id.zoom)
+        panningCheckbox = findViewById(R.id.panning)
+        outdoorCheckbox = findViewById(R.id.outdoor)
+        val streetViewPanoramaFragment =
+            supportFragmentManager.findFragmentById(R.id.streetviewpanorama) as SupportStreetViewPanoramaFragment?
+        streetViewPanoramaFragment?.getStreetViewPanoramaAsync { panorama: StreetViewPanorama ->
+            streetViewPanorama = panorama
+            streetViewPanorama?.isStreetNamesEnabled = streetNameCheckbox.isChecked()
+            streetViewPanorama?.isUserNavigationEnabled = navigationCheckbox.isChecked()
+            streetViewPanorama?.isZoomGesturesEnabled = zoomCheckbox.isChecked()
+            streetViewPanorama?.isPanningGesturesEnabled = panningCheckbox.isChecked()
```

Comment:
Nit: reference `panorama` here instead so you can call the property directly vs. having a `?.` safety check.

Reply:
Sounds good! I changed in https://github.com/googlemaps/android-samples/pull/208/commits/556a374a03ee123386384b1b6e2040683b4c5a06, and made Java match.